src/gui/transferlistdelegate.cpp

```diff
@@ -70,6 +70,11 @@ void TransferListDelegate::paint(QPainter * painter, const QStyleOptionViewItem
     QStyleOptionViewItem opt = QItemDelegate::setOptions(index, option);
     QItemDelegate::drawBackground(painter, opt, index);
     switch (index.column()) {
+    case TorrentModel::TR_NAME:
+    case TorrentModel::TR_SAVE_PATH:
```

Comment:
>The cases below look better to me.

They are correct. You misunderstood Coding Guidelines. We do not insist on the use of brackets unless they are required (e.g. when some **case** has local variables).